src/SPIRV/Silk.NET.SPIRV/Enums/ImageChannelOrder.gen.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System;
+using Silk.NET.Core.Attributes;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.SPIRV
+{
+    [NativeName("Name", "SpvImageChannelOrder_")]
+    public enum ImageChannelOrder : int
+    {
+        [NativeName("Name", "SpvImageChannelOrderR")]
+        R = 0x0,
+        [NativeName("Name", "SpvImageChannelOrderA")]
+        A = 0x1,
+        [NativeName("Name", "SpvImageChannelOrderRG")]
+        RG = 0x2,
+        [NativeName("Name", "SpvImageChannelOrderRA")]
+        RA = 0x3,
+        [NativeName("Name", "SpvImageChannelOrderRGB")]
+        Rgb = 0x4,
+        [NativeName("Name", "SpvImageChannelOrderRGBA")]
+        Rgba = 0x5,
+        [NativeName("Name", "SpvImageChannelOrderBGRA")]
+        Bgra = 0x6,
+        [NativeName("Name", "SpvImageChannelOrderARGB")]
+        Argb = 0x7,
+        [NativeName("Name", "SpvImageChannelOrderIntensity")]
+        Intensity = 0x8,
+        [NativeName("Name", "SpvImageChannelOrderLuminance")]
+        Luminance = 0x9,
+        [NativeName("Name", "SpvImageChannelOrderRx")]
+        Rx = 0xA,
+        [NativeName("Name", "SpvImageChannelOrderRGx")]
+        RGx = 0xB,
+        [NativeName("Name", "SpvImageChannelOrderRGBx")]
+        RGBx = 0xC,
+        [NativeName("Name", "SpvImageChannelOrderDepth")]
+        Depth = 0xD,
+        [NativeName("Name", "SpvImageChannelOrderDepthStencil")]
+        DepthStencil = 0xE,
+        [NativeName("Name", "SpvImageChannelOrdersRGB")]
```

Comment:
This is incorrectly trimmed to be Rgb (instead of Srgb) so 2 things:
1. Can we figure out why the current trimming logic isn't stopping at Order vs Orders (as our current code should just think they are different prefixes)
2. Can we bring back prefix overrides to account for this

Reply:
I'm not sure why its not stopping, and yeah prefix overrides would work here, although we'd need another special case for "if its in the prefix overrides, then we use the native name as the trimming name as before instead of lenient underscore"

Reply:
Yeah that's fine, if it's a prefix override we take that as gospel including the casing, rather than going the underscore paths, converting back to pascal case after through the naming translator